docs/src/paradox/03-apis/api-v2/reading-and-searching-resources.md

```diff
@@ -61,6 +61,124 @@ See the interfaces `Resource` and `ResourcesSequence` in module
 `ResourcesResponse` (exists for both API schemas: `ApiV2Simple` and
 `ApiV2WithValueObjects`).
 
+### Text Markup Options
+
+Text markup can be returned in one of two ways:
+
+- As XML embedded in the response, using an @ref:[XML to Standoff Mapping](xml-to-standoff-mapping.md).
+
+- As @ref:[standoff/RDF](../../02-knora-ontologies/knora-base.md#text-with-standoff-markup),
+  which is Knora's internal markup representation.
+
+Embedded XML is the default.
+
+Implementation of support for standoff/RDF in API v2 is in its early stages. The basic
+procedure works like this:
+
+First, request a resource in the @ref:[complex schema](introduction.md#api-schema), using any relevant
+API v2 route, submitting the string `standoff` as the value of either:
+
+  - the HTTP header `X-Knora-Accept-Markup`
+
+  - the URL parameter `markup`
+
+If a text value in the resource contains markup, the text value will look something like this:
+
+```jsonld
+{
+  "@id" : "http://rdfh.ch/0001/LK-wKXDNQJaRHOf0F0aJ2g/values/1Er1OpVwQR2u6peTwyNpJw",
+  "@type" : "knora-api:TextValue",
+  "knora-api:attachedToUser" : {
+    "@id" : "http://rdfh.ch/users/9XBCrDV3SRa7kS1WwynB4Q"
+  },
+  "knora-api:hasPermissions" : "CR knora-admin:Creator|V knora-admin:UnknownUser",
+  "knora-api:textValueHasMarkup" : true,
+  "knora-api:textValueHasMaxStandoffStartIndex" : 6737,
+  "knora-api:userHasPermission" : "CR",
+  "knora-api:valueAsString" : "\nHamlet\nACT I\nSCENE I. Elsinore. A platform before the castle...",
+  "knora-api:valueCreationDate" : {
+    "@type" : "xsd:dateTimeStamp",
+    "@value" : "2019-05-08T17:08:32.158401Z"
+  }
+}
+```
+
+The object `knora-api:valueAsString` contains the text without markup. The predicate
+`knora-api:textValueHasMarkup` indicates that the text value has markup,
+and the value of the predicate `knora-api:textValueHasMaxStandoffStartIndex` gives the start
+index of the last standoff tag; this gives the client some idea of how much markup there is.
+
+You can then request the text value's standoff/RDF, which is returned in pages of a limited
+size. To get each page:
+
+```
+HTTP GET to http://host/v2/standoff/RESOURCE_IRI/TEXT_VALUE_IRI/OFFSET
```

Comment:
I had to provide `?markup=standoff` as a param to make it work in the browser (did not modify any headers). In this case (`v2/standoff/resIri/textValIri`), shouldn't standoff be the default? What else would the client want to get when using this route?

Reply:
You're right, I'll fix that.

Reply:
Done in fe74ba6.